src/GenerateCommand.php

```diff
@@ -37,11 +38,15 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $this->getComposer()->getConfig()->merge([
             'scripts' => [
                 "phpcs" => "vendor/bin/phpcs",
+                "php-cs-fixer" => "vendor/bin/php-cs-fixer fix --ansi",
+                "php-cs-fixer-ci" => "vendor/bin/php-cs-fixer fix --dry-run --ansi",
                 "phpcbf" => "vendor/bin/phpcbf",
                 "phpstan" => "vendor/bin/phpstan analyse",
                 "rector-ci" => "vendor/bin/rector process --dry-run --ansi",
                 "rector" => "vendor/bin/rector process --ansi",
             ]
         ]);
+
+        return 0;
```

Comment:
Without this, a type error will be thrown on future versions of Composer.
```
Return value of "Worksome\CodingStyle\GenerateCommand::execute()" must be of the type int, "null" returned.
```

Reply:
Ah makes sense 👍